spec/annotator-content-analysis_spec.js

```diff
@@ -1,6 +1,42 @@
+const options = {
+  dataUrl: '../demo/tua.json',
+  templates: {
+    form: '/templates/form.handlebars',
+    topic: '/templates/topic.handlebars',
+    question: '/templates/question.handlebars',
+    answer: '/templates/answer.handlebars'
+  }
+};
+
+let firstRange = new Range({
+  end: '/div[1]',
+  endOffset: 24,
+  start: '/div[1]',
+  startOffset: 0
+});
+
+const annotationFixture = {
+  quote: 'Santa Cruz city attorney',
+  ranges: [firstRange]
+};
+
 describe('Annotator Custom Editor ', function() {
-  return describe('On loading the module', function() {
-    return it('should set the data', function() {
+  let analyzer = annotatorContentAnalysis(options);
+  analyzer.start();
+
+  describe('On loading the module', () => {
+    it('exists', () => expect(analyzer).to.exist);
+  });
+
+  describe('has the methods we want', () => {
+    it('should set the data', () => {
+      // TODO: create an annotation fixture, pass it into analyzer.beforeAnnotationCreated
+      // doing so now causes the module to log the survey to the page
+      // this is the first affirmation needed to decouple data presentation from data generation
+      console.log('options.dataUrl', analyzer.beforeAnnotationCreated(annotationFixture));
```

Comment:
I still think this is good progress. Let's not spin our wheels too much on this for the time being. I'm going to put an ask in to the annotatator dev board and see what suggestions they have.